Dockerfile

```
@@ -5,6 +5,7 @@ USER root
 RUN apt-get -y -qq update \
  && apt-get -y -qq install \
         dbus-x11 \
+        xclip \
```

This is needed for the clipboard test. We could add a condition build arg so this isn't installed by default, at the expense of making the build and test more complicated.

---

I think its okay that we add it. I think having a comment could be nice.

```suggestion
        # xclip is only added because its required by the jupyter-remote-desktop-proxy's tests
        xclip \
```